src/pip/_internal/operations/install/wheel.py

```diff
@@ -618,6 +620,30 @@ def is_entrypoint_wrapper(name):
     shutil.move(temp_record, record)
 
 
+def install_wheel(
+    name,  # type: str
+    wheel_path,  # type: str
+    scheme,  # type: Scheme
+    req_description,  # type: str
+    pycompile=True,  # type: bool
+    warn_script_location=True,  # type: bool
+    _temp_dir_for_testing=None,  # type: Optional[str]
```

Comment:
It feels kinda wrong: if we want to test wheels with a dist-info directory containing empty directory (i.i. `test_dist_info_contains_empty_dir` I'd rather we use a modified wheel file...

Reply:
See #8548. :)